apps/playground/src/app/modules/components/checkbox-module/checkbox-module.component.html

```diff
@@ -28,87 +29,117 @@ <h2>DsfrCheckboxComponent</h2>
   <dl class='input-properties'>
     <play-input-property-definition
       name='legend'
+      required='Required only if a group of several checkboxes is to be displayed'
       type='It takes a string'>
-      It is required.
+      This property represents the text displayed on top of a whole group of checkboxes and enables the end-user to know
+      how the checkboxes in this group are related and what they are about.
     </play-input-property-definition>
     <play-input-property-definition
       name='legendId'
+      required='Required only if a group of several checkboxes is to be displayed'
       type='It takes a string'>
-      It is required.
+      The ID used for the <code>legend</code> HTML tag .
     </play-input-property-definition>
     <play-input-property-definition
       name='hint'
       type='It takes a string'>
+      An optional text to help end-users understand the purpose of a group of checkboxes. It is a way to give more
+      details and complement the legend. Only displayed for a group of several checkboxes.
     </play-input-property-definition>
     <play-input-property-definition
       name='name'
       type='It takes a string'>
+      The value for the <code>name</code> attribute of the checkboxes.
     </play-input-property-definition>
     <play-input-property-definition
       name='items'
-      type='It takes an array of objects with the format {id: string, label: string, value: string, hint?: string}'>
-      It is required.
+      [required]='true'
+      type='CheckboxItem[]'>
+      The value must be an array of objects satisfying the <code>CheckboxItem</code> interface : each object must have
+      an <code>id</code> property, a <code>label</code> property and a <code>value</code> property. It may also have an
+      optional <code>hint</code> property. All these properties have string values.
```

Comment:
`It may also have an optional <code>hint</code> property. All these properties have string values. Another one, the optional <code>disabled</code> property will take a boolean.`